content/2_Bronze/Intro_Sets.mdx

```diff
@@ -442,46 +439,41 @@ One way is to get around this is to create a new map.
 
 ```py
 d = {i: i for i in range(10)}
-d_new = dict(item for i, item in enumerate(d.items()) if i % 3 != 2)
+d_new = dict(item for i, item in enumerate(d.items()) if [condition])
```

Comment:
why have this
like the `[condition]` part

Reply:
ok, ill add a comment or smth explaining that it removes every third element

Reply:
wait what

Reply:
?

Reply:
no i meant like i see you replaced the condition with just `[condition]`
i was thinking you should leave that be

Reply:
oh, i meant that i'd revert the change and add a comment to explain (since i removed the original explanation from the first code block)

Reply:
uhhh sure ig
i'll see when you actually do it

Reply:
added the changes. 

java still uses a set for the other two examples, i think it's fine, but lmk if you think it should be changed.